tests/services/analysis_starter/test_analysis_starter_factory.py

```diff
@@ -43,6 +51,42 @@ def test_analysis_starter_factory_microsalt(cg_context: CGConfig, mocker: Mocker
     assert isinstance(analysis_starter.submitter, SubprocessSubmitter)
 
 
+def test_analysis_starter_factory_mip_dna():
+    # GIVEN a CGConfig with a MIP-DNA config
```

Comment:
I think this removes a bit of name-clashing
```suggestion
    # GIVEN a CGConfig with a configuration info for MIP-DNA
```